### PR TITLE
fixing sort, dedicated javascript file for the function

### PR DESCRIPTION
**Pull request by madoleary**
* [x] Are you working on the latest release?
* [x] Have you tested it locally?
* [->] Please explain your change

This ought to fix outstanding issues with the sorting mechanism on the services index page, referenced in issue #822 . 

Thanks !

### CODE REVIEW

File: app/assets/javascripts/services/table.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  }
+  if (elementB.dataset.classification === 'F') {
+    return 1;
+  }
+  if (elementA.dataset.classification === 'N/A') {
+    return -1;
+  }
+  if (elementB.dataset.classification === 'N/A') {
+    return 1;
+  }
```
Comment (JesseWeinstein) [inline app/assets/javascripts/services/table.js]: I strongly suspect there is a shorter way to write this... :-)

**Comment by madoleary** [comment]
You could make a PR :)

**Comment by JesseWeinstein** [comment]
I plan to. :-)

File: app/assets/javascripts/services/table.js (hunk 1 above)
Comment (Vinnl) [reply app/assets/javascripts/services/table.js]: Do make sure that, if you submit a PR, the shorter way is also at least as readable as the current way - code is read more often than it is written! :)

**Pull request merged**